packages/components/src/components/Menu/Menu.stories.tsx

```diff
@@ -61,13 +62,44 @@ const CustomButtonMenu = (): JSX.Element => {
 
   return (
     <Box.div minHeight="100px">
-      <Menu items={items} menuButton={menuButton} />
+      <Menu gutter={3} items={items} menuButton={menuButton} />
     </Box.div>
   );
 };
 
 export const WithCustomMenuButton: Story = {
-  render: (): JSX.Element => <CustomButtonMenu />,
+  render: (): React.JSX.Element => <CustomButtonMenu />,
+};
+
+export const WithCustomStoreAndMenuButtonProps: Story = {
+  render: (): React.JSX.Element => {
+    const store = useMenuStore({
+      placement: "right-start",
+    });
+
+    const items: MenuItemProps[] = [
+      {
+        onClick: noop,
+        label: "Option 1",
+      },
+      {
+        onClick: noop,
+        label: "Option 2",
+      },
+    ];
+
+    return (
+      <Box.div minHeight="100px">
+        <Menu
+          customStore={store}
+          items={items}
+          menuButtonProps={{
```

Comment:
I didn't want to create another story only to show the change in the menuButtonProps.